firoplugin/src/main/java/hec/firoplugin/FIROEnsemblePlugin.java

```diff
@@ -0,0 +1,96 @@
+package hec.firoplugin;
+
+import hec.externalplugin.ExternalDataLocation;
+import hec.externalplugin.ExternalDataType;
+import hec.externalplugin.SelfRegisteringExternalDataPlugin;
+
+import javax.swing.JFileChooser;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class FIROEnsemblePlugin extends SelfRegisteringExternalDataPlugin
```

Comment:
Consider FIRO_External_EnsemblePlugin or FIRO_ResSim_EnsemblePlugin? I dont really want to get it confused with WAT or cwms plugins, because this is not a wat or cwms plugin.  Not critical to change, but it might reduce confusion?

Reply:
Good ideas.   Help me choose between your options above . Is this only used by ResSim in current design?

Reply:
As far as I understand only in ResSim in the current design (possibly also in the ressim plugin for wat, but i am not so clear on that)

Reply:
using FIRO_ResSim_EnsemblePlugin